app/controllers/cart_controller.rb

```diff
@@ -61,4 +65,14 @@ def check_remain_availability
       cart_detail.quantity > cart_detail.product.remain_quantity
     end
   end
+
+  def handle_update_success current_cart_detail, quantity
+    if check_quantity quantity
+      current_cart_detail.update(quantity:)
+    else
+      current_cart_detail.destroy
+    end
+    flash[:success] = t "cart.update_success"
+    redirect_to cart_path
```

Comment:
Huynh thử như này có gọn hơn k á
```suggestion
def handle_update_success(current_cart_detail, quantity)
  current_cart_detail.update(quantity:) if check_quantity(quantity) || current_cart_detail.destroy
  flash[:success] = t "cart.update_success"
  redirect_to cart_path
end
```

Reply:
oke

Reply:
nó bị lỗi nha